src/index.js

```diff
@@ -1 +1,35 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const tasksList = [
+  { index: 0, completed: false, description: 'Debug weather app code' },
+  { index: 1, completed: false, description: 'Prepare for StandUp meeting' },
+  { index: 2, completed: true, description: 'Study documentation about WebPack' },
+  { index: 3, completed: false, description: 'Organize folders' },
+  { index: 4, completed: false, description: 'Finish ToDo List' },
+];
+
+function createTaskCards() {
+  const taskContainer = document.querySelector('.task-container');
+  const taskTemplate = document.getElementById('task-template');
+  tasksList.forEach((task) => {
+    const taskCard = taskTemplate.content.cloneNode(true).children[0];
```

Comment:
- [ ] From the requirement of this project, we are asked to render tasks dynamically to the DOM on page load but here, you are cloning a template. Please let's create an HTML template in this file and render it.

Reply:
Hello @FrederickMih . Sorry but I do not understand this very well, could you elaborate a little more?
I am rendering dynamically the tasks, as you can see in the index.html, it just contains the template to use, then when the page loads, I use this template to generate the task dynamically from the array of objects. Thank you

Reply:
Hi @VicPeralta , What I meant here is that the HTML template that gets the input from the user ie task-description should be created inside the index.js file.